### ISSUE TITLE
Simple set up question

### ISSUE THREAD

**Issue by leskat47** [EndBug/version-check#108]
I think perhaps I've misunderstood something in the set up. 

Error I'm getting:
<img width="762" alt="Screen Shot 2021-09-17 at 11 55 53 AM" src="https://user-images.githubusercontent.com/10586810/133839940-054d4089-4694-476e-883f-c50306d34157.png">

My set up is pretty minimal in my github action:
```
    name: Version check
        id: check
        uses: EndBug/version-check@v1.1.0
        with:
          diff-search: true
```

I see that the documentation says:
`when someone changes the version in package.json to 1.2.3 and pushes a commit with the message <WHATEVER> 1.2.3 (eg. Release 1.2.3 or Bump version to v1.2.3), output values are set `

This particular commit didn't have a change to the version, but I assume that shouldn't cause the error. The 404 error suggests to me that I have something set up incorrectly. 

I'd appreciate any advice.

**Comment by EndBug** [comment]
In version [`v1.1.0`](https://github.com/EndBug/version-check/tree/v1.1.0) private repos are supported, but you need to manually pass the token through the `token` input.
Anyway, I suggest you update the action to the latest version, which automatically fills that input.

**Comment by leskat47** [comment]
Thanks for the quick response. I'm seeing the same issue with the latest version, unfortunately.

**Comment by leskat47** [comment]
Sure. It is private, but here are the logs:

![Screen Shot 2021-09-22 at 3 20 16 PM](https://user-images.githubusercontent.com/10586810/134429871-09ec8ab5-db17-4a99-bbd2-20b5098fc655.png)
![Screen Shot 2021-09-22 at 3 21 10 PM](https://user-images.githubusercontent.com/10586810/134429868-b2947cfe-a403-4f93-9c05-649d293c395f.png)
![Screen Shot 2021-09-22 at 3 21 59 PM](https://user-images.githubusercontent.com/10586810/134429862-7252d13e-2fb2-46b8-8842-0b25aa21da05.png)

**Comment by EndBug** [comment]
Sorry, but I don't see my action in the screenshots you sent...
Also, please attach both the **text logs** (gear icon > view raw logs) **AND the workflow file** that has been used, otherwise I'll not be able to help you.

To create a spoiler, use these tags:
```
<details>
<summary>The title of the spoiler</summary>

Other text
</details>
```
<details>
<summary>The title of the spoiler</summary>

Other text
</details>

**Comment by leskat47** [comment]
Sorry about that. 

<details><summary>Here are the raw logs:</summary>
2021-09-20T19:44:39.1089746Z Found online and idle hosted runner in the current repository's organization account that matches the required labels: 'ubuntu-latest'
2021-09-20T19:44:39.1750727Z Waiting for a Hosted runner in the 'organization' to pick this job...
2021-09-20T19:44:40.0497966Z Job is waiting for a hosted runner to come online.
2021-09-20T19:44:44.3014629Z Job is about to start running on the hosted runner: GitHub Actions 8 (hosted)
2021-09-20T19:44:48.9288154Z Current runner version: '2.282.1'
2021-09-20T19:44:48.9314048Z ##[group]Operating System
2021-09-20T19:44:48.9315008Z Ubuntu
2021-09-20T19:44:48.9315455Z 20.04.3
2021-09-20T19:44:48.9315862Z LTS
2021-09-20T19:44:48.9316291Z ##[endgroup]
2021-09-20T19:44:48.9316837Z ##[group]Virtual Environment
2021-09-20T19:44:48.9317387Z Environment: ubuntu-20.04
2021-09-20T19:44:48.9317900Z Version: 20210913.1
2021-09-20T19:44:48.9318755Z Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20210913.1/images/linux/Ubuntu2004-README.md
2021-09-20T19:44:48.9320037Z Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20210913.1
2021-09-20T19:44:48.9320835Z ##[endgroup]
2021-09-20T19:44:48.9321529Z ##[group]Virtual Environment Provisioner
2021-09-20T19:44:48.9322344Z 1.0.0.0-master-20210816-1
2021-09-20T19:44:48.9322821Z ##[endgroup]
2021-09-20T19:44:48.9325026Z ##[group]GITHUB_TOKEN Permissions
2021-09-20T19:44:48.9326643Z Actions: write
2021-09-20T19:44:48.9327408Z Checks: write
2021-09-20T19:44:48.9327891Z Contents: write
2021-09-20T19:44:48.9328485Z Deployments: write
2021-09-20T19:44:48.9329160Z Discussions: write
2021-09-20T19:44:48.9329672Z Issues: write
2021-09-20T19:44:48.9330266Z Metadata: read
2021-09-20T19:44:48.9330770Z Packages: write
2021-09-20T19:44:48.9331271Z PullRequests: write
2021-09-20T19:44:48.9331873Z RepositoryProjects: write
2021-09-20T19:44:48.9332479Z SecurityEvents: write
2021-09-20T19:44:48.9332961Z Statuses: write
2021-09-20T19:44:48.9333611Z ##[endgroup]
2021-09-20T19:44:48.9336516Z Prepare workflow directory
2021-09-20T19:44:48.9993963Z Prepare all required actions
2021-09-20T19:44:49.0007638Z Getting action download info
2021-09-20T19:44:49.1962270Z Download action repository 'actions/checkout@v2' (SHA:5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f)
2021-09-20T19:44:51.1006991Z Download action repository 'EndBug/version-check@v2.0.1' (SHA:5f74c25ac6da99d9fef740efce54e012d5146e89)
2021-09-20T19:44:51.5252908Z ##[group]Run actions/checkout@v2
2021-09-20T19:44:51.5253673Z with:
2021-09-20T19:44:51.5254223Z   repository: captable/aurora
2021-09-20T19:44:51.5255241Z   token: ***
2021-09-20T19:44:51.5255716Z   ssh-strict: true
2021-09-20T19:44:51.5256319Z   persist-credentials: true
2021-09-20T19:44:51.5256838Z   clean: true
2021-09-20T19:44:51.5257326Z   fetch-depth: 1
2021-09-20T19:44:51.5257770Z   lfs: false
2021-09-20T19:44:51.5258292Z   submodules: false
2021-09-20T19:44:51.5258812Z ##[endgroup]
2021-09-20T19:44:52.0904079Z Syncing repository: captable/aurora
2021-09-20T19:44:52.0905753Z ##[group]Getting Git version info
2021-09-20T19:44:52.0907068Z Working directory is '/home/runner/work/aurora/aurora'
2021-09-20T19:44:52.0908393Z [command]/usr/bin/git version
2021-09-20T19:44:52.0909116Z git version 2.33.0
2021-09-20T19:44:52.0911342Z ##[endgroup]
2021-09-20T19:44:52.0912679Z Deleting the contents of '/home/runner/work/aurora/aurora'
2021-09-20T19:44:52.0915229Z ##[group]Initializing the repository
2021-09-20T19:44:52.0916038Z [command]/usr/bin/git init /home/runner/work/aurora/aurora
2021-09-20T19:44:52.0917089Z hint: Using 'master' as the name for the initial branch. This default branch name
2021-09-20T19:44:52.0918003Z hint: is subject to change. To configure the initial branch name to use in all
2021-09-20T19:44:52.0918917Z hint: of your new repositories, which will suppress this warning, call:
2021-09-20T19:44:52.0919530Z hint: 
2021-09-20T19:44:52.0920487Z hint: 	git config --global init.defaultBranch <name>
2021-09-20T19:44:52.0921060Z hint: 
2021-09-20T19:44:52.0921852Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2021-09-20T19:44:52.0922920Z hint: 'development'. The just-created branch can be renamed via this command:
2021-09-20T19:44:52.0923545Z hint: 
2021-09-20T19:44:52.0924177Z hint: 	git branch -m <name>
2021-09-20T19:44:52.0924867Z Initialized empty Git repository in /home/runner/work/aurora/aurora/.git/
2021-09-20T19:44:52.0925805Z [command]/usr/bin/git remote add origin https://github.com/captable/aurora
2021-09-20T19:44:52.0927330Z ##[endgroup]
2021-09-20T19:44:52.0928454Z ##[group]Disabling automatic garbage collection
2021-09-20T19:44:52.0929326Z [command]/usr/bin/git config --local gc.auto 0
2021-09-20T19:44:52.0930438Z ##[endgroup]
2021-09-20T19:44:52.0933039Z ##[group]Setting up auth
2021-09-20T19:44:52.0934005Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2021-09-20T19:44:52.0935447Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2021-09-20T19:44:52.0936969Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2021-09-20T19:44:52.0938766Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2021-09-20T19:44:52.0940532Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2021-09-20T19:44:52.0941901Z ##[endgroup]
2021-09-20T19:44:52.0942947Z ##[group]Fetching the repository
2021-09-20T19:44:52.0944826Z [command]/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +e4366697ac851b42c6a74f0b9438b3108dce749c:refs/remotes/origin/main
2021-09-20T19:44:52.1981503Z remote: Enumerating objects: 55, done.        
2021-09-20T19:44:52.1983138Z remote: Counting objects:   1% (1/55)        
2021-09-20T19:44:52.1984149Z remote: Counting objects:   3% (2/55)        
2021-09-20T19:44:52.1984792Z remote: Counting objects:   5% (3/55)        
2021-09-20T19:44:52.1985368Z remote: Counting objects:   7% (4/55)        
2021-09-20T19:44:52.1987960Z remote: Counting objects:   9% (5/55)        
2021-09-20T19:44:52.1988551Z remote: Counting objects:  10% (6/55)        
2021-09-20T19:44:52.1989542Z remote: Counting objects:  12% (7/55)        
2021-09-20T19:44:52.1990216Z remote: Counting objects:  14% (8/55)        
2021-09-20T19:44:52.1990926Z remote: Counting objects:  16% (9/55)        
2021-09-20T19:44:52.1991530Z remote: Counting objects:  18% (10/55)        
2021-09-20T19:44:52.1992434Z remote: Counting objects:  20% (11/55)        
2021-09-20T19:44:52.1993066Z remote: Counting objects:  21% (12/55)        
2021-09-20T19:44:52.1993653Z remote: Counting objects:  23% (13/55)        
2021-09-20T19:44:52.1994212Z remote: Counting objects:  25% (14/55)        
2021-09-20T19:44:52.1994854Z remote: Counting objects:  27% (15/55)        
2021-09-20T19:44:52.1995555Z remote: Counting objects:  29% (16/55)        
2021-09-20T19:44:52.1996120Z remote: Counting objects:  30% (17/55)        
2021-09-20T19:44:52.1996645Z remote: Counting objects:  32% (18/55)        
2021-09-20T19:44:52.1997230Z remote: Counting objects:  34% (19/55)        
2021-09-20T19:44:52.1997830Z remote: Counting objects:  36% (20/55)        
2021-09-20T19:44:52.1998527Z remote: Counting objects:  38% (21/55)        
2021-09-20T19:44:52.1999990Z remote: Counting objects:  40% (22/55)        
2021-09-20T19:44:52.2000693Z remote: Counting objects:  41% (23/55)        
2021-09-20T19:44:52.2001444Z remote: Counting objects:  43% (24/55)        
2021-09-20T19:44:52.2002228Z remote: Counting objects:  45% (25/55)        
2021-09-20T19:44:52.2003349Z remote: Counting objects:  47% (26/55)        
2021-09-20T19:44:52.2003971Z remote: Counting objects:  49% (27/55)        
2021-09-20T19:44:52.2004557Z remote: Counting objects:  50% (28/55)        
2021-09-20T19:44:52.2005176Z remote: Counting objects:  52% (29/55)        
2021-09-20T19:44:52.2005746Z remote: Counting objects:  54% (30/55)        
2021-09-20T19:44:52.2006396Z remote: Counting objects:  56% (31/55)        
2021-09-20T19:44:52.2007179Z remote: Counting objects:  58% (32/55)        
2021-09-20T19:44:52.2007893Z remote: Counting objects:  60% (33/55)        
2021-09-20T19:44:52.2008492Z remote: Counting objects:  61% (34/55)        
2021-09-20T19:44:52.2009663Z remote: Counting objects:  63% (35/55)        
2021-09-20T19:44:52.2010263Z remote: Counting objects:  65% (36/55)        
2021-09-20T19:44:52.2011093Z remote: Counting objects:  67% (37/55)        
2021-09-20T19:44:52.2011632Z remote: Counting objects:  69% (38/55)        
2021-09-20T19:44:52.2012215Z remote: Counting objects:  70% (39/55)        
2021-09-20T19:44:52.2012752Z remote: Counting objects:  72% (40/55)        
2021-09-20T19:44:52.2013355Z remote: Counting objects:  74% (41/55)        
2021-09-20T19:44:52.2013905Z remote: Counting objects:  76% (42/55)        
2021-09-20T19:44:52.2014486Z remote: Counting objects:  78% (43/55)        
2021-09-20T19:44:52.2015066Z remote: Counting objects:  80% (44/55)        
2021-09-20T19:44:52.2015601Z remote: Counting objects:  81% (45/55)        
2021-09-20T19:44:52.2016206Z remote: Counting objects:  83% (46/55)        
2021-09-20T19:44:52.2016770Z remote: Counting objects:  85% (47/55)        
2021-09-20T19:44:52.2017413Z remote: Counting objects:  87% (48/55)        
2021-09-20T19:44:52.2018317Z remote: Counting objects:  89% (49/55)        
2021-09-20T19:44:52.2018856Z remote: Counting objects:  90% (50/55)        
2021-09-20T19:44:52.2019473Z remote: Counting objects:  92% (51/55)        
2021-09-20T19:44:52.2020009Z remote: Counting objects:  94% (52/55)        
2021-09-20T19:44:52.2020709Z remote: Counting objects:  96% (53/55)        
2021-09-20T19:44:52.2209616Z remote: Counting objects:  98% (54/55)        
2021-09-20T19:44:52.2210232Z remote: Counting objects: 100% (55/55)        
2021-09-20T19:44:52.2210940Z remote: Counting objects: 100% (55/55), done.        
2021-09-20T19:44:52.2211698Z remote: Compressing objects:   2% (1/45)        
2021-09-20T19:44:52.2212288Z remote: Compressing objects:   4% (2/45)        
2021-09-20T19:44:52.2212875Z remote: Compressing objects:   6% (3/45)        
2021-09-20T19:44:52.2213446Z remote: Compressing objects:   8% (4/45)        
2021-09-20T19:44:52.2214242Z remote: Compressing objects:  11% (5/45)        
2021-09-20T19:44:52.2215017Z remote: Compressing objects:  13% (6/45)        
2021-09-20T19:44:52.2215605Z remote: Compressing objects:  15% (7/45)        
2021-09-20T19:44:52.2216168Z remote: Compressing objects:  17% (8/45)        
2021-09-20T19:44:52.2216720Z remote: Compressing objects:  20% (9/45)        
2021-09-20T19:44:52.2217282Z remote: Compressing objects:  22% (10/45)        
2021-09-20T19:44:52.2217848Z remote: Compressing objects:  24% (11/45)        
2021-09-20T19:44:52.2218396Z remote: Compressing objects:  26% (12/45)        
2021-09-20T19:44:52.2218961Z remote: Compressing objects:  28% (13/45)        
2021-09-20T19:44:52.2219505Z remote: Compressing objects:  31% (14/45)        
2021-09-20T19:44:52.2220060Z remote: Compressing objects:  33% (15/45)        
2021-09-20T19:44:52.2220620Z remote: Compressing objects:  35% (16/45)        
2021-09-20T19:44:52.2221170Z remote: Compressing objects:  37% (17/45)        
2021-09-20T19:44:52.2221733Z remote: Compressing objects:  40% (18/45)        
2021-09-20T19:44:52.2222282Z remote: Compressing objects:  42% (19/45)        
2021-09-20T19:44:52.2222830Z remote: Compressing objects:  44% (20/45)        
2021-09-20T19:44:52.2223371Z remote: Compressing objects:  46% (21/45)        
2021-09-20T19:44:52.2223927Z remote: Compressing objects:  48% (22/45)        
2021-09-20T19:44:52.2224483Z remote: Compressing objects:  51% (23/45)        
2021-09-20T19:44:52.2225025Z remote: Compressing objects:  53% (24/45)        
2021-09-20T19:44:52.2225571Z remote: Compressing objects:  55% (25/45)        
2021-09-20T19:44:52.2226116Z remote: Compressing objects:  57% (26/45)        
2021-09-20T19:44:52.2226678Z remote: Compressing objects:  60% (27/45)        
2021-09-20T19:44:52.2227224Z remote: Compressing objects:  62% (28/45)        
2021-09-20T19:44:52.2227772Z remote: Compressing objects:  64% (29/45)        
2021-09-20T19:44:52.2228320Z remote: Compressing objects:  66% (30/45)        
2021-09-20T19:44:52.2228872Z remote: Compressing objects:  68% (31/45)        
2021-09-20T19:44:52.2229568Z remote: Compressing objects:  71% (32/45)        
2021-09-20T19:44:52.2230118Z remote: Compressing objects:  73% (33/45)        
2021-09-20T19:44:52.2230677Z remote: Compressing objects:  75% (34/45)        
2021-09-20T19:44:52.2231240Z remote: Compressing objects:  77% (35/45)        
2021-09-20T19:44:52.2231789Z remote: Compressing objects:  80% (36/45)        
2021-09-20T19:44:52.2232707Z remote: Compressing objects:  82% (37/45)        
2021-09-20T19:44:52.2233277Z remote: Compressing objects:  84% (38/45)        
2021-09-20T19:44:52.2233847Z remote: Compressing objects:  86% (39/45)        
2021-09-20T19:44:52.2234411Z remote: Compressing objects:  88% (40/45)        
2021-09-20T19:44:52.2234971Z remote: Compressing objects:  91% (41/45)        
2021-09-20T19:44:52.2235692Z remote: Compressing objects:  93% (42/45)        
2021-09-20T19:44:52.2236240Z remote: Compressing objects:  95% (43/45)        
2021-09-20T19:44:52.2236799Z remote: Compressing objects:  97% (44/45)        
2021-09-20T19:44:52.2237354Z remote: Compressing objects: 100% (45/45)        
2021-09-20T19:44:52.2237925Z remote: Compressing objects: 100% (45/45), done.        
2021-09-20T19:44:52.2416868Z remote: Total 55 (delta 1), reused 35 (delta 1), pack-reused 0        
2021-09-20T19:44:52.2595569Z From https://github.com/captable/aurora
2021-09-20T19:44:52.2597800Z  * [new ref]         e4366697ac851b42c6a74f0b9438b3108dce749c -> origin/main
2021-09-20T19:44:52.2653259Z ##[endgroup]
2021-09-20T19:44:52.2654339Z ##[group]Determining the checkout info
2021-09-20T19:44:52.2655358Z ##[endgroup]
2021-09-20T19:44:52.2656295Z ##[group]Checking out the ref
2021-09-20T19:44:52.2658389Z [command]/usr/bin/git checkout --progress --force -B main refs/remotes/origin/main
2021-09-20T19:44:52.2765545Z Switched to a new branch 'main'
2021-09-20T19:44:52.2767496Z Branch 'main' set up to track remote branch 'main' from 'origin'.
2021-09-20T19:44:52.2774586Z ##[endgroup]
2021-09-20T19:44:52.2815631Z [command]/usr/bin/git log -1 --format='%H'
2021-09-20T19:44:52.2852495Z 'e4366697ac851b42c6a74f0b9438b3108dce749c'
2021-09-20T19:44:52.3046950Z ##[group]Run EndBug/version-check@v2.0.1
2021-09-20T19:44:52.3047564Z with:
2021-09-20T19:44:52.3048025Z   diff-search: true
2021-09-20T19:44:52.3048496Z ##[endgroup]
2021-09-20T19:44:52.3673066Z Searching for version update...
2021-09-20T19:44:52.3703375Z ##[group]Searching in 1 commit...
2021-09-20T19:44:52.3712778Z Package file name: "package.json"
2021-09-20T19:44:52.3713680Z Package file URL: undefined
2021-09-20T19:44:52.3714441Z Version assumptions: undefined
2021-09-20T19:44:52.3715685Z ##[endgroup]
2021-09-20T19:44:52.3716903Z No standard npm version commit found, switching to diff search (this could take more time...)
2021-09-20T19:44:52.3723128Z ##[group]Checking the diffs of 1 commit...
2021-09-20T19:44:52.4649043Z ##[error]An error occurred in checkDiff:
Error: Request failed with status code 404
2021-09-20T19:44:52.4667229Z ##[error]Error: Command failed with code 1
2021-09-20T19:44:52.4697986Z ##[group]Outputs
2021-09-20T19:44:52.4698557Z changed: false
2021-09-20T19:44:52.4699722Z ##[endgroup]
2021-09-20T19:44:52.4781151Z Post job cleanup.
2021-09-20T19:44:52.6129872Z [command]/usr/bin/git version
2021-09-20T19:44:52.6133590Z git version 2.33.0
2021-09-20T19:44:52.6147093Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2021-09-20T19:44:52.6150119Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2021-09-20T19:44:52.6302652Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2021-09-20T19:44:52.6323132Z http.https://github.com/.extraheader
2021-09-20T19:44:52.6332132Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2021-09-20T19:44:52.6368695Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2021-09-20T19:44:52.6625579Z Cleaning up orphan processes
</details>

Or the file, if that's easier to read:
[logs.txt](https://github.com/EndBug/version-check/files/7219078/logs.txt)

And here is the workflow:
```
name: release
on:
  push:
    branches:
      - main

jobs:
  check:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout repository
        uses: actions/checkout@v2
      - name: Version check
        id: check
        uses: EndBug/version-check@v2.0.1
        with:
          diff-search: true
  publish:
    name: Build and Publish
    runs-on: ubuntu-latest
    needs: check
    if: ${{ needs.check.outputs.changed == 'true' }}
    steps:
      - name: Set node version
        uses: actions/setup-node@v2
        with:
          node-version: '12.x'
          registry-url: 'https://npm.pkg.github.com'
      - name: Get repository
        uses: actions/checkout@v2
      - name: Build
        run: yarn build
      - name: Publish
        run: yarn publish
        env:
          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by EndBug** [comment]
I'm sorry, my bad: I told you that the current version automatically gets the token, but that change hasn't been released (I thought I'd include it in the rewrite, but then forgot about it for months 😅)

So, try using the `token` input:
```yaml
- name: Version check
   id: check
   uses: EndBug/version-check@v2.0.1
     with:
       diff-search: true
       token: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by leskat47** [comment]
Thanks @EndBug! That was the solution. I appreciate your help. :)
